index.html

```diff
@@ -0,0 +1,136 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+
+    <title>Document</title>
+
+</head>
+<body>
+  <div class="blurry"></div>
+    
+  <div class="firstheader">
+    <h1 class="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+  </div>
+  
+  <div class="content">
+        <div class="top-box">
+
+              <div class="topbox-content"> 
+                
+                
+                <form action="" id="survey-form">
+
+                <label for="" id="name-label" >Name</label>
+                <input type="text" placeholder="Enter your name" id="name" >
+            <br>
+
+                <label for="" id="email-label">Email</label>
+                <input type="text" placeholder="Enter your Email"id="email" >
+            <br>
+                <label for="" id="number-label">Age(optional)</label>
+                <input type="number" placeholder="Age" min="0" max="70" id="number">
+            <br>
+
+
+                
+            <label for="">Which option best describes your current role?</label><br>
+
+            <select name="Most Like" id="drop-down" style=margin-bottom:30px;>
+              <option value="" disabled selected hidden>Select a current role</option>
+              <option value="Student">Student</option>
+              <option value="Full Time Job">Full Time Job</option>
+              <option value="Community">Full Time Learner</option>
+              <option value="Open Source">Prefer not to say</option>
+              <option value="Open Source">Other</option>
+
+            </select>
+            <br>
+
+
+
+            <label for="">Would you recommend freecodecamp to a friend</label> <br>
+            <input type="radio" name="freecode" style=margin-bottom:30px;display:inline><span>Definitely</span> <br>
+            <input type="radio" name="freecode"  style=margin-bottom:30px;display:inline><span>Maybe</span> <br>
+            <input type="radio" name="freecode" style=margin-bottom:30px;display:inline><span>Not Sure</span> <br>
+
+
+            <br>
+
+
+            <label for="">What is your favorite freecodecamp feature?</label><br>
+
+            <select name="Most Like" id="most-like" style=margin-bottom:30px;>
+              <option value="" disabled selected hidden>Select an option</option>
+              <option value="Challenges">Challenges</option>
+              <option value="Projects">Projects</option>
+              <option value="Community">Community</option>
+              <option value="Open Source">Open Source</option>
+
+            </select>
+
+            <br>
+
+            <label for="">What would you like to see improved? (check all that applies)</label> <br>
+
+            <input type="checkbox" style="display:inline">
+            <span>Front-end Projects</span>
+            <br>
+
```

Comment:
In order to avoid linter errors, please kindly reduce the whitespaces. It would help reduce the bugs/errors found in your code and save you time.